package.json

```diff
@@ -6,6 +6,7 @@
   "author": "Michal Ochman",
   "license": "MIT",
   "repository": "github:michalochman/react-pixi-fiber",
+  "typings": "./index.d.ts",
```

Comment:
shouldn't `index.d.ts` also be included in `files` so npm/yarn downloads it when installing?

Reply:
Good catch, I'll add this.

Reply:
@olivoil published as `v0.2.4`